skbio/stats/composition.py

```diff
@@ -642,6 +645,133 @@ def ilr_inv(mat, basis=None, check=True):
     return clr_inv(np.dot(mat, basis))
 
 
+@experimental(as_of="0.5.1-dev")
+def alr(mat, denominator_idx=0):
+    r"""
+    Performs additive log ratio transformation.
+
+    This function transforms compositions from a D-part Aitchison simplex to
+    a non-isometric real space of D-1 dimensions. The argument
+    `denominator_col` defines the index of the column used as the common
+    denominator. The :math: `alr` transformed data are amenable to multivariate
+    analysis as long as statistics don't involve distances.
+
+    :math:`alr: S^D \rightarrow \mathbb{R}^{D-1}`
+
+    The alr transformation is defined as follows
+
+    .. math::
+        alr(x) = [\ln \frac{x_1}{x_D}, \ldots, \ln \frac{x_{D-1}}{x_D}]
+
+    where :math:`D` is the index of the part used as common denominator.
+
+    Parameters
+    ----------
+    mat: numpy.ndarray
+       a matrix of proportions where
+       rows = compositions and
+       columns = components
+
+    denominator_idx: int
+       the index of the column (2D-matrix) or position (vector) of
+       mat which should be used as the reference composition
+
+    Returns
+    -------
+    numpy.ndarray
+         alr-transformed data projected in a non-isometric real space
+         of D-1 dimensions for a D-parts composition
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skbio.stats.composition import alr
+    >>> x = np.array([.1, .3, .4, .2])
+    >>> alr(x)
+    array([ 1.09861229,  1.38629436,  0.69314718])
+    """
+    mat = closure(mat)
+    if len(mat.shape) == 2:
+        mat_t = mat.T
+        numerator_idx = list(range(0, mat_t.shape[0]))
+        del numerator_idx[denominator_idx]
+        lr = np.log(mat_t[numerator_idx, :]/mat_t[denominator_idx, :]).T
+    elif len(mat.shape) == 1:
+        numerator_idx = list(range(0, mat.shape[0]))
+        del numerator_idx[denominator_idx]
+        lr = np.log(mat[numerator_idx]/mat[denominator_idx])
+    else:
+        raise ValueError("mat must be either 1D or 2D")
+    return lr
+
+
+@experimental(as_of="0.5.1-dev")
+def alr_inv(mat, denominator_idx=0):
+    r"""
+    Performs inverse additive log ratio transform.
+
+    This function transforms compositions from the non-isometric real space of
+    alrs to Aitchison geometry.
+
+    :math:`alr^{-1}: \mathbb{R}^{D-1} \rightarrow S^D`
+
+    The inverse alr transformation is defined as follows
+
+    .. math::
+         alr^{-1}(x) = C[exp([y_1, y_2, ..., y_{D-1}, 0])]
+
+    where :math:`C[x]` is the closure operation defined as
+    .. math::
+        C[x] = \left[\frac{x_1}{\sum_{i=1}^{D} x_i},\ldots,
+                     \frac{x_D}{\sum_{i=1}^{D} x_i} \right]
```

Comment:
sphinx threw an indentation error here.  I'd try to throw in an extra line.

I'd also recommend checking out the documentation docs [here](https://github.com/biocore/scikit-bio/tree/master/doc) on how to run sphinx and render the html pages.